src/components/DatePickers/DatePicker.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import './DatePicker.scss';
+import shortid from 'shortid';
+
+
+export const DatePicker = ({ text }) => {
```

Comment:
There is something we could do better in this component. I bet you finding it out :)